krm-functions/Makefile

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile for in-tree functions.
+
+# Build all in-tree functions locally
+build-local:
```

Comment:
Ideally we should provide a way for user to build one function. WDYT?

Reply:
That's a good idea. I've added a way for users to do it: `FUNCTION_NAME=render-helm-chart make build-local`